ansible/deploy.yml

```diff
@@ -108,7 +108,9 @@
     - role: dashd
       tags:
         - dashd
-    - elastic_beats
+    - role: elastic_beats
+      when:
+        - inventory_hostname not in hp_masternodes
```

Comment:
We are working on `hosts: miners` here, this condition (and other conditions) are not necessary